frontend/src/employee-mobile-frontend/generated/api-clients/daycare.ts

```diff
@@ -536,6 +536,22 @@ export async function createPreschoolTerm(
 }
 
 
+/**
+* Generated from fi.espoo.evaka.daycare.controllers.TermsController.deletePreschoolTerm
+*/
+export async function deletePreschoolTerm(
```

Comment:
This unnecessary duplicate generation under the `employee-mobile-frontend` package could be avoided by specifying the user type more strictly in controller methods:
`user: AuthenticatedUser` --> `user: AuthenticatedUser.Employee`

Reply:
Nice, I added this change and regenerated endpoints.